lib/Dialect/Comb/CombOps.cpp

```diff
@@ -282,6 +282,18 @@ LogicalResult ExtractOp::verify() {
   return success();
 }
 
+LogicalResult TruthTableOp::verify() {
+  size_t numInputs = getInputs().size();
+  if (numInputs >= sizeof(size_t) * 8)
+    return emitOpError("Truth tables support a maximum of ")
+           << sizeof(size_t) * 8 - 1 << " inputs on your platform";
+
+  ArrayAttr table = getLookupTable();
+  if (table.size() != (1ull << numInputs))
+    return emitOpError("Expected lookup table of 2^n length");
```

Comment:
nit: `emitOpError("Expected lookup table of 2^n length") << ", n = " << numInputs;`